sql/2018-04-17-subscribers-table.sql

```diff
@@ -0,0 +1,12 @@
+-- This file creates a new table, subscription, in the ejabberd DB.
+
+USE ejabberd;
+
+CREATE TABLE subscription (
+    jid varchar(50) NOT NULL,
+    room varchar(255) NOT NULL,
+    last_updated timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
ts standard is `date_created` unless there's a reason for this to be different

Reply:
Ejabberd standard has `created_at` and I was following that for most of these

Reply:
kk